README.md

```diff
@@ -677,6 +677,14 @@ https://twitter.com/fhinkel
 Topics: JavaScript (TC39), V8, Chrome\
 https://twitter.com/mathias
 
+### Saarbrücken
+
+<img src="https://avatars.githubusercontent.com/u/357884?v=4" height="70px" width="70px" align="left" alt="" />
```

Comment:
we use the Twitter avatar in the avatars folder - so the way would be:

1. upload the image below to the avatars folder with the name `ThorstenHans` (matching your Twitter handle, no file extension)
2. set the `src` to `./avatars/ThorstenHans` (no file extension)

![ThorstenHans](https://github.com/karlhorky/awesome-speakers/assets/1935696/479744c3-2266-42fa-88a4-5251f791381e)